src/test/java/org/elasticsearch/bwcompat/OldIndexBackwardsCompatibilityTests.java

```diff
@@ -439,13 +439,11 @@ void assertDeleteByQueryWorked(String indexName, Version version) throws Excepti
     }
 
     void assertUpgradeWorks(String indexName, boolean alreadyLatest) throws Exception {
-        HttpRequestBuilder httpClient = httpClient();
-
         if (alreadyLatest == false) {
-            UpgradeTest.assertNotUpgraded(httpClient, indexName);
+            UpgradeTest.assertNotUpgraded(client(), indexName);
```

Comment:
You can remove the http enabled in settings in this class now? Thank you for this! I think it should make these tests a bit faster!